deno/lib/types.ts

```diff
@@ -619,6 +620,26 @@ function isValidIP(ip: string, version?: IpVersion) {
   return false;
 }
 
+function isValidJwt(token: string) {
+  try {
+    const tokensParts = token.split(".");
+    if (tokensParts.length !== 3) {
+      return false;
+    }
+
+    const [header] = tokensParts;
+    const parsedHeader = JSON.parse(Buffer.from(header, "base64").toString());
```

Comment:
It should works both in node.js and modern browsers, so you can't use `Buffer`. You can take a look at the code. https://github.com/validatorjs/validator.js/blob/master/src/lib/isJWT.js

Reply:
Sorry, not a front guy 🙈. Replace `Buffer` for `atob`. Also add option to pass algorithm used in JWT